specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -9186,7 +9186,7 @@
           ],
           "x-ms-enum": {
             "name": "SecurityTypes",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
This change needs to be made to the latest Compute version, 2021-03-01, not this older version.
Older versions are not used to generate SDKs.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/9a9e414dbc002d8d2f2bcb754dfba306f084ef01/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

Reply:
Did the changes in the new commit that I have pushed in api version 2021-03-01